src/accounts/accounts.service.ts

```diff
@@ -116,4 +118,35 @@ export class AccountsService {
       webhookUrl,
     });
   }
+
+  public async revokePermissions(params: RevokePermissionsDto): ServiceReturn<void> {
+    const { signer, webhookUrl, secondaryAccounts } = params;
+
+    const { revokePermissions } = this.polymeshService.polymeshApi.accountManagement;
+
+    return this.transactionsService.submit(
+      revokePermissions,
+      {
+        secondaryAccounts,
```

Comment:
Here the `secondaryAccounts` is `string[]`. I forgot to update the procedure definition for which I raised a separate [PR #882](https://github.com/PolymeshAssociation/polymesh-sdk/pull/882). Once it gets merged, I'll bump up the SDK version and build will then pass. (I changed it locally and have already tested it)